tfexec/cmd.go

```diff
@@ -275,3 +303,97 @@ func writeOutput(ctx context.Context, r io.ReadCloser, w io.Writer) error {
 		}
 	}
 }
+
+func (tf *Terraform) runTerraformCmd(ctx context.Context, cmd *exec.Cmd) error {
```

Comment:
Merged the runTerraformCmd implementation in `cmd_default.go` an `cmd_linux.go` into one. The only difference is the different value for `cmd.SysProcAttr` at line 310.